src/components/TabsPanel.tsx

```diff
@@ -59,89 +58,89 @@ const useStyles = makeStyles((theme: Theme) => ({
       "& .MuiButtonBase-root:nth-child(2)": {
         backgroundColor: "rgba(255, 255, 255, 1)",
       },
-      "& .MuiButtonBase-root.chip-1": {
-        background: `linear-gradient(90deg, rgba(255,255,255,0.5) 25%, ${COLORS[0]} 100%)`,
+      "& .MuiButtonBase-root.chip-b3e5fc": {
```

Comment:
Maybe the COLORS const could have been just moved here to `TabsPanel.tsx` and those changes would not be that needed.